src/libs/portfolio/helpers.ts

```diff
@@ -446,3 +446,15 @@ export const processTokens = (
     return tokens
   }, [] as TokenResult[])
 }
+
+export const shouldShowConfettiLogic = (accountState: AccountState, resBalance: any[]) => {
```

Comment:
Too broad func name. Either add a comment on what these confetti are for, or name this better (or both :) )